source/deployment/bleve.rst

```diff
@@ -1,21 +1,30 @@
 Bleve Search (Experimental)
 ===========================
 
-Bleve is a search engine that uses Lucene-style full-text search and indexing. This style of search and indexing helps overcome limitations of the default database search such as challenges with characters and advanced search capabilities. 
+Bleve is a search engine that uses Lucene-style full-text search and indexing. This style of search and indexing helps overcome limitations of the default database search such as challenges with characters and advanced search capabilities.
 
 The Bleve search engine works as a library integrated into the Mattermost codebase. As it generates indexes in the filesystem of the server that it is running on, it doesn’t require an external server to function. Because of this, Bleve should not be enabled in High Availability deployments.
 
 Configuring Bleve in Mattermost
-------------------------------
+-------------------------------
 
 Follow these steps to configure the Mattermost server to use Bleve and generate required indexes. Once the configuration is saved, new posts made to the database will be automatically indexed with Bleve.
 
 **Note:** During indexing, search results may be incomplete until the indexing job is complete.
 
 1. Open **System Console > Experimental > Bleve**.
-2. Set **Enable Bleve Indexing** to **true** to enable the other settings on the page. 
+2. Set **Enable Bleve Indexing** to **true** to enable the other settings on the page.
 3. Set the directory path to use for storing bleve indexes (e.g.: ``/var/opt/mattermost/bleveindexes``). The user running Mattermost should have permissions to access the directory.
 4. Save the configuration.
-5. Click the **Index Now** button. All users, channels, and posts in the database will be indexed oldest to newest.  
+5. Click the **Index Now** button. All users, channels, and posts in the database will be indexed oldest to newest.
 6. Set **Enable Bleve for search queries** to **true**.
 7. Set **Enable Bleve for autocomplete queries** to **true**.
+
+Bleve Search
+------------
+
+The following conditions are applied when using Bleve search:
+
+* **Unquoted terms:** Search terms that contain non-alphanumeric characters/special characters outside of quotation marks are removed. For example, using ``abcd "**" && abc`` as a search term will return results for a search for ``abcd "**" abc`` as the ``&&`` characters weren't within the quotation marks.
+* **Wildcard search:** Wildcard search (e.g., ``abc*``) is supported.
+* **Excluded words:** The following stop words are excluded from search in MySQL: ``"a", "about", "an", "are", "as", "at", "be", "by", "com", "de", "en", "for", "from", "how", "i", "in", "is", "it", "la", "of", "on", "or", "that", "the", "this", "to", "was", "what", "when", "where", "who", "will", "with", "und", "the", "www"``.
```

Comment:
Recommend this go in https://docs.mattermost.com/help/getting-started/searching.html#technical-notes since it is specifically about MySQL and not Bleve.

Reply:
Agreed. I've removed this and moved it. New PR has been created for the change.